shallow_backup/reinstall.py

```diff
@@ -116,8 +125,8 @@ def reinstall_packages_sb(packages_path):
 		elif pm == "gem":
 			print_red_bold("WARNING: Gem reinstallation is not supported.")
 		elif pm == "cargo":
-			print_red_bold("WARNING: Cargo reinstallation is not possible at the moment."
-			                 "\n -> https://github.com/rust-lang/cargo/issues/5593")
+			print_red_bold("WARNING: Cargo reinstallation is not possible at the moment.\
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['print_red_bold' may be undefined, or defined from star imports: .compatibility, .printing](https://app.codacy.com/manual/alichtman/shallow-backup/pullRequest?prid=4896077)